doc/maintenance.md

```diff
@@ -0,0 +1,539 @@
+- MUST respond with `404` if the resource was not found;
+
+##### Response
+
+The `SubmitStructureResponse` message must be returned in any case (success, partial success, failure).
+This is defined as part of the `RegistryInterface` messages.
+The details of the message are explained in section **Structure response message**, below.
+
+The following matrix summarises the returned `HTTP` response codes.
+
+| Method | Exists | Versioning respected | Is Referenced | References exist/provided | Return Code |
+|---|---|---|---|---|---|
+| PUT/POST | T | T | F | T | `200` (successful) or `207` (partially successful)  |
+| PUT/POST | T | T | F | F | `409` |
+| PUT/POST | T | T | T | T | `200` if update is possible - `409` if references would break |
+| PUT/POST | T | F | - | - | `409` in case of changes that violate the versioning requirements |
+| PUT/POST | - | - | - | - | `422` see section **Server** |
+| PUT | F | - | - | - | `404` |
+
+<sup>T: True, F: False, I: Irrelevant, -: Not applicable</sup>
+
+#### Partially update Item Schemes
+
+In the case of Item Schemes, it is possible to submit a partial Item Scheme, including the Items that must be updated.
+The usage of `POST` and `PUT` is the same as in the case of full replacement. The only difference is that the submitted Items Schemes to be partially updated (i.e. not completely replaced) must include the flag `isPartial="true"`.
+
+In this case, any Item included in the submitted partial Item Scheme:
+- fully replaces the Item, if it existed in the stored Item Scheme; the Item stays in the same position;
+- is added as a new Item, if it did not exist
+  - in a selected position, if it is submitted relatively to existing Items;
+  - else, at the end of the Item Scheme.
+
+For more details on nested Items (like Categories), please refer to section **Nested Items** below.
+
+In addition to the Items, the Item Scheme details are also updated accordingly, i.e. the names, description, annotations and any other attributes.
+Names and Descriptions are replaced if they exist for the submitted language (i.e. the value of their `lang` attribute), or simply added, if new. Annotations and other Item Scheme attributes are fully replaced.
+
+##### Client
+
+In order to partially update an Item Scheme, the client:
+
+- MAY set the `Accept` header to indicate the preferred response format;
+- MUST set the `Content-type` header according to the format of the submitted Item Scheme(s);
+- MUST include the flag `isPartial="true"` in the submitted Item Scheme(s);
+- MUST include in the request body:
+  - For `PUT`, **one** Item Scheme in the SDMX format indicated in the `Content-type` header and of the Item Scheme type indicated in the resource, e.g. a Concept Scheme under resource `/conceptscheme`;
```

Comment:
According to the HTML standard, for PUT you must use exactly the same resource URL as you would expect to use for the subsequent GET to get exactly the same input back as result. Thus to update only specific items A, B and C in an itemscheme OECD:CL(1.0.0), you must put the request under /OECD/CL/1.0.0/A,B,C (see itemID path parameter). It's the same behaviour as is implemented for the DELETE action. Also here, if you want to delete items A, B and C the resource URL to use is /OECD/CL/1.0.0/A,B,C.